packages/components/src/Link/Link.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { PropsWithChildren } from "react";
+
+interface LinkProps {
+  readonly external?: boolean;
+  readonly url: string;
+}
+
```

Comment:
This implementation wraps `<Link>` around `children` but the storybook and test files seem to use a different, self closing tag version of `<Link />` that takes a "title" prop. Would be good to pick one and be consistent. (The tests could just be scrapped for now or treated as WIP with a comment. But the story would be good to update with the proposed approach.)